docs/config.md

```diff
@@ -34,8 +35,8 @@ rdap:
 
         - name: arin
           servers:
-              - https://rdap.arin.net/registry
-              - http://rdap.arin.net/registry
+              - https://rdap.arin.net/registry/
```

Comment:
No they don't need to be terminated by forward clash. Not sure where or why this change has occurred.